src/code42cli/cmds/devices.py

```diff
@@ -0,0 +1,399 @@
+_DEVICE_INFO_KEYS_MAP["osHostname"] = "Hostname"
+_DEVICE_INFO_KEYS_MAP["guid"] = "GUID"
+_DEVICE_INFO_KEYS_MAP["status"] = "Status"
+_DEVICE_INFO_KEYS_MAP["archiveBytes"] = "Largest Archive Size in Bytes"
+_DEVICE_INFO_KEYS_MAP["lastConnected"] = "Last Connected Date"
+_DEVICE_INFO_KEYS_MAP["lastBackup"] = "Last Backup Date"
+_DEVICE_INFO_KEYS_MAP["lastCompletedBackup"] = "Last Completed Backup Date"
+_DEVICE_INFO_KEYS_MAP["productVersion"] = "Code42 Version"
+_DEVICE_INFO_KEYS_MAP["osName"] = "Operating System"
+_DEVICE_INFO_KEYS_MAP["osVersion"] = "Operating System Version"
+
+
+@devices.command()
+@device_id_option
+@format_option
+@sdk_options()
+def show(state, device_id, format=None):
+    """Print device info."""
+    formatter = OutputFormatter(format, _DEVICE_INFO_KEYS_MAP)
+    device_info = _get_device_info(state.sdk, device_id)
+    if device_info:
+        formatter.echo_formatted_list([device_info])
+
+
+def _get_device_info(sdk, device_id):
+    device = sdk.devices.get_by_id(device_id, include_backup_usage=True).data
+    device["archiveBytes"] = (
+        max(
+            [
+                backupDestination["archiveBytes"]
+                for backupDestination in device["backupUsage"]
+            ]
+        )
+        if len(device["backupUsage"]) > 0
+        else 0
+    )
+    device["lastBackup"] = (
+        max(
+            [
+                backupDestination["lastBackup"]
+                for backupDestination in device["backupUsage"]
+            ]
+        )
+        if len(device["backupUsage"]) > 0
+        else None
+    )
+    device["lastCompletedBackup"] = (
+        max(
+            [
+                backupDestination["lastCompletedBackup"]
+                for backupDestination in device["backupUsage"]
+            ]
+        )
+        if len(device["backupUsage"]) > 0
+        else None
+    )
+    return device
+
+
+@devices.group(cls=OrderedGroup)
+@sdk_options(hidden=True)
+def bulk(state):
+    """Tools for managing devices in bulk"""
+    pass
+
+
+@bulk.command(name="list", help="Get information about many devices")
+@click.option(
+    "--active",
+    required=False,
+    type=bool,
+    is_flag=True,
+    default=None,
+    help="Include to return only active devices.",
+)
+@click.option(
+    "--days-since-last-connected",
+    required=False,
+    type=int,
+    help="Return only devices that have not connected in the number of days specified.",
+)
+@click.option(
+    "--org-uid",
+    required=False,
+    type=str,
+    default=None,
+    help="Optionally provide to limit devices to only the ones in the org you specify. Note that child orgs will be included.",
+)
+@click.option(
+    "--drop-most-recent",
+    required=False,
+    type=int,
+    help="Will drop the X most recently connected devices for each user from the result list where X is the number you provide as this argument. Can be used to avoid passing the most recently connected device for a user to the deactivate command",
```

Comment:
I know our formatters might not be bothered by these longer help texts, but if you manually hit `Enter` for it to be multiline, our formatter with automatically adjust it but still keep it multi-line, if that makes sense. It would be nice!  This feedback applies to all new help texts for click options in this PR

Reply:
also missing period on this one